packages/builder/src/pages/builder/auth/_components/OIDCButton.svelte

```diff
@@ -29,9 +30,9 @@
     }
   })
 
-  $: src = !$oidc.logo
+  $: src = !oidcLogoCheck
```

Comment:
What was the purpose of pulling out `$oidc.logo` to a separate variable but then only using it once in the statement? Personally I think that if something can't be simplified to a one liner, then multi-line ternary statements should be broken out into functions to make it easier to understand the logic, and possible to maintain it in future.